apps/aecore/lib/aecore/structures/signed_tx.ex

```diff
@@ -5,11 +5,12 @@ defmodule Aecore.Structures.SignedTx do
 
   alias Aecore.Keys.Worker, as: Keys
   alias Aecore.Structures.SpendTx
+  alias Aecore.Structures.DataTx
   alias Aecore.Structures.SignedTx
   alias Aeutil.Serialization
 
   @type t :: %SignedTx{
-    data: SpendTx.t(),
+    data: SpendTx.t() | DataTx.t(),
```

Comment:
can you add a `@type data_types :: SpendTx.t() | DataTx.t()`

Reply:
Maybe this will be obsolete after the suggested changes that @rdimitrovbg and me proposed up top are made.